soccer/Robot.cpp

```diff
@@ -265,10 +268,15 @@ float OurRobot::kickTimer() const {
 }
 
 void OurRobot::dribble(uint8_t speed) {
-    uint8_t scaled = *config->dribbler.multiplier * speed;
-    control->set_dvelocity(scaled);
+    Geometry2d::Rect modifiedField = Field_Dimensions::Current_Dimensions.FieldRect() * (*_dribbleOutOfBoundsMult);
+    if (modifiedField.containsPoint(pos)) {
```

Comment:
I'm worried this doesn't actually extend the "playable field" behind our goal line since the y coordinates are 0. Did we check that?